v6/scripts/release.sh

```
@@ -0,0 +1,39 @@
+GH_REPO="$GH_API/repos/$REPO"
+AUTH="Authorization: token ${GITHUB_TOKEN}"
+GH_TAGS="$GH_REPO/releases/tags/$RELEASE"
+payload=$(
+    jq --null-input \
+    --arg tag "$RELEASE" \
+    --arg name "MyEtherWallet $RELEASE" \
+    --arg body "$CHANGELOG" \
+    '{ tag_name: $tag, name: $name, body: $body, draft: false }'
+)
+
+response=$(
```

Shellcheck

[warning] response appears unused. Verify use (or export if used externally).

(SC2034)